.github/workflows/security-scan.yaml

```diff
@@ -117,5 +132,18 @@ jobs:
         with:
           name: dastardly-scan-output
           path: "*dastardly-report.html"
-  
+
+      #uncomment this when work on zap fixes
+      # - name: Check scan exit codes
+      #   run: |
```

Comment:
You can replace all this logic with, which will exit if any of the exit codes are < 0
`run: 
exit $((ZAP_DEMO_EXIT_CODE + ZAP_ACCOUNTS_EXIT_CODE + DASTARDLY_DEMO_EXIT_CODE + DASTARDLY_ACCOUNTS_EXIT_CODE))`